app/helpers/application_helper.rb

```diff
@@ -72,4 +72,20 @@ def content_tag_if(condition, *args, &block)
       capture(&block)
     end
   end
+
+  def admin_loans_path(*args)
+    # Default to active loans, unless overriden
+    options = { loans: { f: { status_value: ['active'] } } }
+    options = args.pop if args.last.is_a? Hash
```

Comment:
Also note that I think the convention on this project is to only put spaces inside curlies if it's a Proc.